src/pyJiraCli/cmd_export.py

```diff
@@ -159,54 +182,53 @@ def _get_filepath(issue, file, path, csv):
                     file_path = path
 
                 else:
-                    return None, Ret.RET_ERROR_WORNG_FILE_FORMAT
+                    ret_status =  Ret.RET_ERROR_WORNG_FILE_FORMAT
             else:
                 # folder to save files was provided
                 if csv:
                     file_path = os.path.join(path, f'{filename}.csv')
                 else:
                     file_path = os.path.join(path, f'{filename}.json')
         else:
-            return None, Ret.RET_ERROR_FILE_NOT_FOUND
+            ret_status = Ret.RET_ERROR_FILE_NOT_FOUND
 
-    return file_path, Ret.RET_OK
+    return file_path, ret_status
 
-def _export_ticket_to_file(issue_key, filepath, user, pw, csv):
-    """"export jira issue from server to json or csv file
+def _export_ticket_to_file(issue_key:str, filepath:str, user:str, pw:str, csv:bool) -> Ret:
```

Comment:
If you specify the type for the return value, it should be done for all to be consistent.
Its up to you ...